audio-download.py

```diff
@@ -95,7 +100,12 @@ def download(q, api, out_folder):
 def download_name(r):
     dt = parsedate(r["recordingDateTime"])
     device_name = r["Device"]["devicename"]
-    ext = "." + MIME_TO_EXT[r["fileMimeType"]]
+
+    mime_type = r.get("fileMimeType")
+    if not mime_type:
+        raise ValueError("recording has no mime type")
```

Comment:
Should that be a KeyError?

Reply:
Nope. The `get` method on a dict returns None by default if the key isn't there. You can optionally pass an alternative default value too.